testdata/data-files/server-side-eval/operators-date-unequal.yml

```diff
@@ -0,0 +1,83 @@
+---
+name: operators - date - unequal
+
+constants:
+  IS_MATCH:
+    value: true
+    variationIndex: 0
+    reason: { kind: "RULE_MATCH", ruleIndex: 0, ruleId: "ruleid" }
+  IS_NOT_MATCH:
+    value: false
+    variationIndex: 1
+    reason: { kind: "FALLTHROUGH" }
+  base_flag: &base_flag
```

Comment:
This `&` syntax, and the `<<: *` references below, isn't a feature of the sdk-test-harness parser; it's just a mixin/macro syntax that's built into YAML.